cv32e40s/tests/programs/custom/pushpop_debug_triggers/pushpop_debug_triggers.c

```diff
@@ -0,0 +1,287 @@
+// Copyright 2023 Silicon Labs, Inc.
```

Comment:
Looks good. At first sight it seems complexed, but when looking deeper into it, it seems like it has the just the complexity it needs (and that it is not that complexed) :-) .